Header header
string name
reachable_occupancy/Capsule[] capsules